cmd/acceptance/main.go

```diff
@@ -114,6 +129,13 @@ func main() {
 			app.Fatalf("failed to install manager: %v", err)
 		}
 
+		for _, runner := range Runners {
+			logger.Log("msg", "running prepare", "runner", reflect.TypeOf(runner).Elem().Name())
+			if err := runner.Prepare(logger, mustClusterConfig()); err != nil {
```

Comment:
you could pull `mustClusterConfig()` outside the for loop, similar to how you did in `run`